app.rb

```diff
@@ -0,0 +1,73 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require 'uri'
+require_relative './database_connection_setup'
+require_relative './lib/peep'
+require_relative './lib/user'
+require_relative './session_helper'
+
+class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+  register Sinatra::Flash
+  include SessionHelper
+
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
+    @user = current_user
+    @peeps = Peep.all
+    erb :peeps
+  end
+
+  get '/peeps/new' do
+    @user = current_user
+    @peeps = Peep.all
+    erb :peep_new
+  end
+
+  post '/peeps' do
+    Peep.create(
+      fullname: current_user.fullname,
+      username: current_user.username,
+      content: params[:content])
+    redirect '/peeps'
+  end
+
+  get '/users/new' do
+    erb :'users_new'
+  end
+
+  post '/users' do
+    user = User.create(email: params['email'],
```

Comment:
I found ActiveRecord really useful for these "creates" with lots of params. I think that as long as the column heading matches the param name it will know which param to put where 😄